plans/tier0.fmf

```diff
@@ -1,89 +1,84 @@
+          how: ansible
+          playbook: tests/ansible_collections/roles/add-custom-repos/main.yml
 
 
 /inhibit_if_kmods_is_not_supported:
-  discover+:
-    test: inhibit-if-kmods-is-not-supported
-  prepare+:
-  - name: copy kmod files to the test machine
-    how: ansible
-    playbook: tests/integration/tier0/inhibit-if-kmods-is-not-supported/ansible/main.yml
+    discover+:
+        test: inhibit-if-kmods-is-not-supported
+    prepare+:
+        - name: copy kmod files to the test machine
+          how: ansible
+          playbook: tests/integration/tier0/inhibit-if-kmods-is-not-supported/ansible/main.yml
 
 
 /inhibit_if_oracle_system_uses_not_standard_kernel:
-  adjust:
-    enabled: false
-    when: >
-      distro != oraclelinux-7 and
-      distro != oraclelinux-8
-  prepare+:
-    - name: install unsupported unbreakable kernel
-      how: ansible
-      playbook: tests/integration/tier0/inhibit-if-oracle-system-uses-not-standard-kernel/ansible/install_unbreakable_kernel.yml
-  discover+:
-    test: inhibit-if-oracle-system-uses-not-standard-kernel
+    adjust+:
+        - enabled: false
+          when: >
+            distro != oraclelinux-7 and
+            distro != oraclelinux-8
+    prepare+:
+        - name: install unsupported unbreakable kernel
+          how: ansible
+          playbook: tests/integration/tier0/inhibit-if-oracle-system-uses-not-standard-kernel/ansible/install_unbreakable_kernel.yml
+    discover+:
+        test: inhibit-if-oracle-system-uses-not-standard-kernel
 
 /releasever_as_mapping:
-  discover+:
-    test: releasever-as-mapping
+    discover+:
+        test: releasever-as-mapping
 
 /check_rollback_handling:
-  discover+:
-    test: test-check-rollback-handling
+    discover+:
+        test: test-check-rollback-handling
 
 /backup_release:
-  discover+:
-    test: backup-release
-  prepare+:
-  - name: install subscription manager
-    how: ansible
-    playbook: tests/ansible_collections/roles/install-submgr/main.yml
+    discover+:
+        test: backup-release
+    prepare+:
+        - name: install subscription manager
+          how: ansible
+          playbook: tests/ansible_collections/roles/install-submgr/main.yml
 
 /check_internet_connection:
-  discover+:
-    test: check-internet-connection
+    discover+:
+        test: check-internet-connection
+
 
-/latest_kernel_check_failed_repoquery:
+/latest_kernel_check:
     discover+:
-        test: verify-latest-kernel-check-with-failed-repoquery
+        test: latest-kernel-check
 
 /sub_man_rollback:
-  adjust:
-    enabled: false
-    when: >
-      distro != centos-8-latest and
-      distro != centos-8.4
-  discover+:
-      test: sub-man-rollback
-
-/custom_kernel:
-  discover+:
-    test: custom-kernel
     adjust+:
-        enabled: false
-        when: >
-            distro != centos-8-latest
+        - enabled: false
+          when: >
+            distro != centos-8-latest and
+            distro != centos-8.4
```

Comment:
can we run it for oracle8 as well?

Reply:
Good idea, I somehow forgot about Oracle Linux latest

Reply:
Oh sorry, I was referring to a different test plan, the subscription-manager issue is related only to CentOS.
Oracle Linux does not have a subscription-manager package available.

Reply:
Oh yes, I thought it was for the `latest_kernel_check`